### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by phieronymus**
Closes #56 
Other documentation for the readme will be added in separate tickets.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,85 +37,146 @@ SDK library for [Node.js][node] to consume [UnivaPay][univapay-url] API.
...
-`univapay-node` requires the following to run:
+### Requirements
 
-  * [Node.js][node] 10 or 12
-  * [npm][npm] (normally comes with Node.js) or [yarn][yarn]
+If you do not already have your store application token, please create it first:
 
+-   Go to the `Store > For developpers > Application tokens` page
+-   Click on `Add`
+-   Add you domain and your mode
```
Comment (laxa88) [inline README.md:67]: `your` typo

File: README.md [hunk 2]
```diff
@@ -40,85 +37,146 @@ SDK library for [Node.js][node] to consume [UnivaPay][univapay-url] API.
...
+try {
+  const transactionToken = await sdk.transactionTokens.create({
+    type: TransactionTokenType.ONE_TIME, // Can be used only once
+    paymentType: PaymentType.CARD,
+
+    // Data type varies depending on the payment type
+    data: { cardholder, cardNumber, expMonth, expYear, cvv }
+  });
+} catch (tokenCreatError: ResponseError) {
+  handleError(errtokenCreatErroror);
```
Comment (laxa88) [inline README.md:107]: errorroror typo

File: README.md [hunk 3]
```diff
@@ -40,85 +37,146 @@ SDK library for [Node.js][node] to consume [UnivaPay][univapay-url] API.
...
+  handleError(chargeCreateError);
+}
 ```
 
-### Customize
+### Polling
 
-You can create your own `SDK` object configuration that only contains selected resources. In the following example show
-how to prepare a class that only has `Stores` resources available:
+After creating a charge the status will rarely directly be `successful` or `failed` and can be `pending` until the API fully processes it. You can poll the charge if you need to know if the charge failed or suceeded.
```
Comment (laxa88) [inline README.md:127]: `rarely` could be subjective, perhaps "After creating a charge, the status will be initialised as `pending`, and eventually be updated to `successful` or `failed`" ?

File: README.md [hunk 4]
```diff
@@ -40,85 +37,146 @@ SDK library for [Node.js][node] to consume [UnivaPay][univapay-url] API.
...
-import { Stores } from "univapay-node/resources/Stores";
+### Create authorization charge
 
-class CustomSDK extends PaymentsSDK {
-    constructor (options?: RestAPIOptions) {
-        super(options);
-        this.stores = new Stores(this.api);
-    }
-}
+By default, creating a charge captures it. If you need to authorize it instead, you can do so passing the `capture` property set to `false` with an optional `captureAt` porpety to automatically capture at a certain date:
```
Comment (laxa88) [inline README.md:161]: `porpety` typo

File: README.md [hunk 5]
```diff
@@ -40,85 +37,146 @@ SDK library for [Node.js][node] to consume [UnivaPay][univapay-url] API.
...
 
-const customSdk = new CustomSDK();
+```typescript
+const charge = await sdk.charges.create({
+  amount: 1000;
+  currency: "JPY",
+  transactionTokenId: transactionToken.id,
+
+  // Do not catpure the charge directly
```
Comment (laxa88) [inline README.md:169]: `catpure` typo 🐱

**Review by laxa88 — APPROVED** [review]

**Comment by phieronymus** [comment]
@laxa88 Thanks a lot for your review 🙂 Also did you feel you could use the SDK to create a charge after reading it or would you have any other questions with just those information?

**Comment by laxa88** [comment]
@phieronymus code samples and explanations looked easy to understand and use. No questions for now 🤔

File: README.md [hunk 6]
```diff
@@ -124,7 +124,7 @@ try {
 
 ### Polling
 
-After creating a charge the status will rarely directly be `successful` or `failed` and can be `pending` until the API fully processes it. You can poll the charge if you need to know if the charge failed or suceeded.
+After creating a charge the status will be initialized to `peding`. It will become `successful` or `failed`after API fully processes it. You can poll the charge if you need to know when the charge bacomes `failed` or `suceeded`:
```
Comment (laxa88) [inline README.md:127]: `pending` typo lol

**Comment by phieronymus** [comment]
Just noticed that the description closes the PR... Closes https://github.com/univapay/univapay-node/issues/54

**Pull request merged**